src/main/java/com/exactpro/th2/common/schema/cradle/CradleConfiguration.java

```diff
@@ -42,6 +42,12 @@
 
     @JsonProperty
     private String cradleInstanceName;
+    
+    @JsonProperty
+    private long timeout;
+    
+    @JsonProperty
+    private int pageSize;
```

Comment:
Please define default values

Reply:
Default value for instance variables for int and long is 0. When they are set to 0 default value inside cradle api is used. Should I set them explicitly?

Reply:
Please define explicitly for understanding